requirements.txt

```
@@ -1,4 +1,4 @@
-arrow>=0.15.6
+arrow==1.0.0
```

Let's be optimistic!

```suggestion
arrow>=1.0.0
```

---

I hope this won't have a catastrophic impact for people installing Watson in their global python distribution...